install-deps.sh

```diff
@@ -243,6 +243,30 @@ else
 		    ceph-libboost-thread1.67-dev \
 		    ceph-libboost-timer1.67-dev
                 ;;
+            *Bionic*)
+                ensure_decent_gcc_on_ubuntu 7 bionic
+                $SUDO add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
+                $SUDO apt-get update
+                install_pkg_on_ubuntu \
```

Comment:
i think we need to rebuild ceph-libboost packages for bionic.

Reply:
Yes. It's better that bionic has its own ceph-libboost packages.
I've verified that xenial version also works.

Reply:
@tchaikov Do you know why "Doc: build check" build failed? I don't know the reason.

Reply:
neither do i. let's re-triggering the doc build to see if it's a transient issue.

Reply:
@tchaikov Thanks. There's one small patch about the way to check gcc version in this PR, I think it needs to be merged.

Reply:
@iotcg could you remove the other commits from this PR, so we can merge it? otherwise we need to wait until i prepare a PR to ready the ceph-libboost packages for bionic.

Reply:
@tchaikov Please check my reply about gcc version check.
Currently, there're below 3 commits in this PR(a/b/c), which one do you want to remove?
a. ubuntu18.04: install ceph-libboost1.67 for ceph build dependencies
https://github.com/ceph/ceph/pull/24776/commits/8666d5010c9ba82411fc546009fc0fda464b8665
b. correct gcc version info as major.minor.patch
https://github.com/ceph/ceph/pull/24776/commits/5e48efb3edbfbfab3c756b62d3bcec404df2b4e9
c. libicu55: set repository to install it on ubuntu18.04
https://github.com/ceph/ceph/pull/24776/commits/19cc0100a1cef744bd357f8fa57ef7aeb96574a3

Reply:
@iotcg i was trying to address your comment

>  There's one small patch about the way to check gcc version in this PR, I think it needs to be merged.

so i'd suggest remove commits other than 5e48efb